src/Summoner/CLI.hs

```diff
@@ -99,6 +136,10 @@ data NewOpts = NewOpts
     , cliConfig   :: PartialConfig  -- ^ config gathered during CLI
     }
 
+data ShowOpts = ShowGhc | ShowLicense LicenseList
```

Comment:
It would be nice to add documentation to this data type and to both it constructors.